### ISSUE TITLE
Sandboxie-Plus UI options to block webcam and microphone

### ISSUE THREAD

**Issue by bastik-1001** [sandboxie-plus/Sandboxie#2130]
### Is your feature request related to a problem or use case?

Looking at #1208 there are some users where webcam and microphone are blocked without it being desired, but having that as a feature appears to be useful.

### Describe the solution you'd like

If it can be done without intending it, it can be blocked on purpose. And if it can be done, there should be UI options to restrict access to webcam and/or microphone. Separately if possible.

### Describe alternatives you've considered

Just having ini options that can be manually set, but such privacy features should not be "hidden".

**Comment by bastik-1001** [comment]
Duplicate of #1669, just the other way around.
